src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/unit/clients/IGDBClientTests.kt

```diff
@@ -0,0 +1,177 @@
+            "    \"artworks\": [\n" +
+            "      {\n" +
+            "        \"id\": 74625,\n" +
+            "        \"url\": \"//images.igdb.com/igdb/image/upload/t_thumb/ar1lkx.jpg\"\n" +
+            "      }\n" +
+            "    ],\n" +
+            "    \"franchises\": [\n" +
+            "      {\n" +
+            "        \"id\": 1932,\n" +
+            "        \"name\": \"Xenoblade\"\n" +
+            "      }\n" +
+            "    ],\n" +
+            "    \"genres\": [\n" +
+            "      {\n" +
+            "        \"id\": 12,\n" +
+            "        \"name\": \"Role-playing (RPG)\"\n" +
+            "      },\n" +
+            "      {\n" +
+            "        \"id\": 31,\n" +
+            "        \"name\": \"Adventure\"\n" +
+            "      }\n" +
+            "    ],\n" +
+            "    \"involved_companies\": [\n" +
+            "      {\n" +
+            "        \"id\": 163667,\n" +
+            "        \"company\": {\n" +
+            "          \"id\": 1119,\n" +
+            "          \"name\": \"Monolith Soft\"\n" +
+            "        }\n" +
+            "      },\n" +
+            "      {\n" +
+            "        \"id\": 163668,\n" +
+            "        \"company\": {\n" +
+            "          \"id\": 70,\n" +
+            "          \"name\": \"Nintendo\"\n" +
+            "        }\n" +
+            "      }\n" +
+            "    ],\n" +
+            "    \"name\": \"Xenoblade Chronicles 3\",\n" +
+            "    \"platforms\": [\n" +
+            "      {\n" +
+            "        \"id\": 130,\n" +
+            "        \"name\": \"Nintendo Switch\"\n" +
+            "      }\n" +
+            "    ],\n" +
+            "    \"release_dates\": [\n" +
+            "      {\n" +
+            "        \"id\": 354757,\n" +
+            "        \"human\": \"Jul 29, 2022\"\n" +
+            "      }\n" +
+            "    ],\n" +
+            "    \"summary\": \"Eunie's the bussss\"\n" +
+            "  }\n" +
+            "]",
+          200
+        )
+      )
+      val result = igdbClient.gamesRequest("test_secret", "191")
+      Assertions.assertEquals(result[0].id, 191411)
+      Assertions.assertEquals(result[0].artworks[0].url, "//images.igdb.com/igdb/image/upload/t_thumb/ar1lkx.jpg")
+      Assertions.assertEquals(result[0].franchises[0].name, "Xenoblade")
+      Assertions.assertEquals(result[0].genres[0].name, "Role-playing (RPG)")
+      Assertions.assertEquals(result[0].genres[1].name, "Adventure")
+      Assertions.assertEquals(result[0].involved_companies[0].company.name, "Monolith Soft")
+      Assertions.assertEquals(result[0].involved_companies[1].company.name, "Nintendo")
+      Assertions.assertEquals(result[0].name, "Xenoblade Chronicles 3")
+      Assertions.assertEquals(result[0].release_dates[0].human, "Jul 29, 2022")
+      Assertions.assertEquals(result[0].summary, "Eunie's the bussss")
+    }
+    @Test
+    fun shouldReturnError() {
```

Comment:
Can we be more verbose in these test names? shouldReturnError when?

Reply:
fixed, might not be a bad idea to discuss as a team best practices for naming tests in general.